ui/playwright.config.ts

```diff
@@ -0,0 +1,16 @@
+import { defineConfig } from 'playwright/test';
+
+export default defineConfig({
+  // Run your local dev server before starting the tests
+  webServer: {
+    command: 'npm run dev',
+    url: 'https://console-api-eyefloaters-dev.mycluster-us-east-107719-da779aef12eee96bd4161f4e402b70ec-0000.us-east.containers.appdomain.cloud',
```

Comment:
This url isn't right, you want to test against localhost:3000, that's the whole purpose of running the dev server with Playwright